src/AcmPage/AcmPage.tsx

```diff
@@ -48,6 +49,8 @@ export function AcmPage(props: { header: ReactNode; children: ReactNode; hasDraw
 export interface AcmPageHeaderProps {
     title: string
     titleTooltip?: string | React.ReactNode
+    label?: string | React.ReactNode
+    labelColor?: 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey'
```

Comment:
Let's not repeat these values here in case they change in PF.

```suggestion
    labelColor?: LabelProps['color']
```